packages/main/src/RPA/Robocorp/Storage.py

```diff
@@ -78,61 +81,229 @@ def print_all_assets():
         return storage.list_assets()
 
     @keyword
-    def get_asset(self, name: str) -> str:
-        """Get an asset's value by providing its `name`.
+    def set_bytes_asset(self, name: str, value: bytes, wait: bool = True):
```

Comment:
Keeping the interface simple with one distinction only between a `value` (no matter the type) and a file `path`.